index.js

```diff
@@ -127,12 +137,19 @@ function wkhtmltopdf(input, options, callback) {
   var stderrMessages = [];
   function handleError(err) {
     var errObj = null;
+    var code;
+    var parallelError;
     if (Array.isArray(err)) {
+      code = err.code;
+      parallelError = err.parallelError;
+      // fix cutted lines in Windows
+      err = Buffer.concat(err).toString();
+      var lines = err.split(/[\r\n]+/).map( line => line.trim() ).filter( line => !!line )
```

Comment:
let's stick to non ES6 syntax for now given that this entire lib wasn't written with ES6